articles/quickstart/native/ios-objc/03-user-sessions.md

```diff
@@ -289,15 +293,11 @@ HybridAuth *auth = [[HybridAuth alloc] init];
 }];
 ```
 
-You can then access its fields as follows:
+Access the user's metadata. You can choose the key names and types for the `user_metadata` dictionary.
```

Comment:
Accessing

Reply:
We try to use imperative mood as it's generally more straightforward